kairon/shared/data/processor.py

```diff
@@ -1117,10 +1118,17 @@ def __extract_actions(self, actions, bot: Text, user: Text):
         saved_actions = self.__prepare_training_actions(bot)
         for action in actions:
             if action.strip().lower() not in saved_actions:
+                self.__check_for_form_and_action_existance(bot, action)
                 new_action = Actions(name=action, bot=bot, user=user)
                 new_action.clean()
                 yield new_action
 
+    def __check_for_form_and_action_existance(self, bot: Text, name: Text):
+        if Utility.is_exist(Actions, raise_error=False, name=name, bot=bot, status=True):
```

Comment:
no need to raise Exception explicitly, use raise_error and exp_message params on is_exist fundtion